core/drivers/unix_socket.h

```diff
@@ -32,6 +32,7 @@
 #define BESS_DRIVERS_UNIXSOCKET_H_
 
 #include <assert.h>
+#include <atomic>
```

Comment:
```
#include <poll.h>

#include <atomic>
#include <cassert>
#include <cerrno>
#include <cstdio>
```

The Google C++ Style Guide says all C++ headers should appear after C headers. Each group needs to be alphabetically sorted.

Do we need `cassert` and `cstdio`? Also, `poll.h` and `cerrno` seem only necessary in the .cc file, not in the header.